src/steamcompmgr.cpp

```diff
@@ -713,6 +717,69 @@ void MouseCursor::setDirty()
 	m_dirty = true;
 }
 
+bool MouseCursor::setCursorImage(char* data, int w, int h)
+{
+    XImage* ximage = XCreateImage(
```

Comment:
Seems like there are mixed tabs and spaces throughout. Also indentation seems off in this function.